### ISSUE TITLE
Don't check for double quoted string in regexp.

### ISSUE THREAD

**Issue by famished-tiger** [rubocop/rubocop#210]
In the Macros4Cuke gem code, I am scanning parts of a Cucumber scenario. String parameters in Cucumber are usually delimited by double quotes. These parameters are detected by the following regular expression:

``` ruby
/"((?:[^\\"]|\\.)*)"/
```

Surprisingly Rubocop complains when it finds this regexp with the following message:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

I expected this message to apply only to string literals not regexps.
